testapi.pm

```diff
@@ -84,13 +84,19 @@ sub check_screen;
 sub type_string;
 sub type_password;
 
-=head1 internal
+=head1 os-autoinst Testapi
+
+This testapi module provides methods exposed by the os-autoinst backend to be used by
+openQA or any other programs that want to interact directly with it.
+
+For more information on how to use openQA, please visit http://open.qa/documentation
```

Comment:
and I shamelessly stole them for https://github.com/os-autoinst/os-autoinst/pull/695